export.md

```diff
@@ -150,7 +152,7 @@ After a bulk data request has been started, a client MAY send a delete request t
 ---
 ### Bulk Data Status Request
 
-After a bulk data request has been started, the client MAY poll the URI provided in the ```Content-Location``` header.  The request MUST include a valid access token in the ```Authorization``` header (i.e., ```Authorization:  Bearer {{token}}```).  See the Security Considerations section above.  
+After a bulk data request has been started, the client MAY poll the URI provided in the ```Content-Location``` header.  All Bulk Data Status Requests MUST include a valid access token in the ```Authorization``` header (i.e., ```Authorization:  Bearer {{token}}```).  See the Security Considerations section above.  
```

Comment:
See https://github.com/smart-on-fhir/fhir-bulk-data-docs/pull/95/files#r241950203. I think we should remove this "must included a bearer token" provision.